js/client-chat-tournament.js

```diff
@@ -357,6 +375,17 @@
 					}
 					break;
 
+				case 'teamlock':
+					if (data[0] === 'on') {
+						this.room.$chat.append("<div class=\"notice tournament-message-teamlock\">This tournament is now team-locked (Tournament players can't change teams between games)</div>");
+						this.teamLocked = true;
+					} else if (data[0] === 'off') {
+						this.room.$chat.append("<div class=\"notice tournament-message-teamlock\">This tournament is no longer team-locked.</div>");
```

Comment:
Why does one of these have a period, but not the other?

Reply:
Because other lines that end in parentheticals don't have a period while the rest do